DOCS.md

```diff
@@ -11,440 +11,397 @@ Version 1.0.0
 
-At this point, the renderer code will take on the responsibility of
-requesting ads from TrueX server, creating the native UI for the TrueX
-choice card and interactive ad unit, as well as communicating events to
-the channel code when action is required.
+At this point, the renderer code will take on the responsibility of requesting ads from true\[X\] server, creating the native UI for the true\[X\] choice card and interactive ad unit, as well as communicating events to the channel code when action is required.
 
-The channel code will still need to parse out the SSAI ad response,
-detect when a TrueX ad is supposed to display, pause the stream,
-instantiate `TruexAdRenderer` and handle any events emitted by the
-renderer code.
+The channel code will still need to parse out the SSAI ad response, detect when a true\[X\] ad is supposed to display, pause the stream, instantiate `TruexAdRenderer` and handle any events emitted by the renderer code.
+
+It will also need to handle skipping ads in the current ad pod, if it is notified to do so.
 
-It will also need to handle skipping ads in the current ad pod, if it is
-notified to do so.
 
 ## Product Flows
 
-There are two distinct product flows supported by `TruexAdRenderer`:
-Sponsored Stream (full-stream ad-replacement) and Sponsored Ad Break
-(mid-roll ad-replacement).
+There are two distinct product flows supported by `TruexAdRenderer`: Sponsored Stream (full-stream ad-replacement) and Sponsored Ad Break (mid-roll ad-replacement).
 
-In a Sponsored Ad Break flow, once the user hits a mid-roll break with a
-TrueX tag flighted, they will be shown a "choice-card" offering them the
-choice between watching a normal set of video ads or a fully interactive
-TrueX ad:
+In a Sponsored Ad Break flow, once the user hits a mid-roll break with a true\[X\] tag flighted, they will be shown a "choice-card" offering them the choice between watching a normal set of video ads or a fully interactive true\[X\] ad:
 
 ![choice card](media/choice_card.png)
 
 ***Fig. A** example true\[X\] mid-roll choice card*
 
-If the user opts for a normal ad break, or if the user does not make a
-selection before the 30 second countdown timer expires, the TrueX UI
-will close and playback of normal video ads can continue as usual.
+If the user opts for a normal ad break, or if the user does not make a selection before the countdown timer expires, the true\[X\] UI will close and playback of normal video ads can continue as usual.
 
-If the user opts to interact with TrueX, an interactive ad unit will be
-shown to the user:
+If the user opts to interact with TrueX, an interactive ad unit will be shown to the user:
```

Comment:
Nit, find and replace instances of `TrueX,`

Reply:
ty